migrations/versions/689a6ce963c3_remove_anonymous_columns.py

```diff
@@ -0,0 +1,33 @@
+"""remove anonymous columns
+
+Revision ID: 689a6ce963c3
+Revises: c79589787f2f
+Create Date: 2022-03-08 21:27:44.689439
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '689a6ce963c3'
+down_revision = 'c79589787f2f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    """
+    Giving niceties anonymously in bulk is a partially implemented
+    feature for the user.
+
+    Ultimately want to discourage this feature as well as make it easier for
+    code maintenance.
+    """
+    op.drop_column("user", "anonymous_by_default")
+    op.drop_column("nicety", "anonymous")
```

Comment:
I think we do still want to allow anonymous niceties! Just not saving the default preference.